test/integration/isolated/examples_udproute_test.go

```diff
@@ -0,0 +1,61 @@
+//go:build integration_tests
+
+package isolated
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/kong/kubernetes-testing-framework/pkg/clusters"
+	"github.com/stretchr/testify/assert"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+
+	"github.com/kong/kubernetes-ingress-controller/v2/test/integration/consts"
+	"github.com/kong/kubernetes-ingress-controller/v2/test/internal/helpers"
+)
+
+func TestExampleUDPRoute(t *testing.T) {
+	t.Parallel()
+
+	udpRouteExampleManifests := fmt.Sprintf("%s/gateway-udproute.yaml", examplesDIR)
+
+	f := features.
+		New("example").
+		WithLabel("example", "true").
+		WithLabel("gatewayapi", "alpha").
+		WithLabel("kind", "UDPRoute").
```

Comment:
We could define consts for labels and expect at least subset of them to be always set. I believe we should have them at least for the "dimensions" that we'd use in CI for separating tests.

Reply:
I thought about that but didn't add it in the first pass. Good call.

Adding it now.